README.md

```diff
@@ -102,6 +102,25 @@ async function eventHandler(payload, headers, inputs) {
 }
 ```
 
+The `as_json` can be used to get a json payload instead of a `Uint8Array`:
+
+```javascript
+const { memphis } = require("memphis-functions");
+
+exports.handler = async (event) => {
+    return await memphis.createFunction(event, eventHandler, as_json = true);
+};
+
+async function eventHandler(payload, headers, inputs) {
+    payload["modified"] = true;
+
+    return {
+        processedMessage: Buffer.from(JSON.stringify(payload), 'utf-8'),
```

Comment:
in case it is a json, the user should not convert it to bytes as well we can do it for him